pkg/operator/certrotation/signer.go

```diff
@@ -92,6 +92,7 @@ func (c RotatedSigningCASecret) EnsureSigningCertKeyPair(ctx context.Context) (*
 			reason = "secret doesn't exist"
 		}
 		c.EventRecorder.Eventf("SignerUpdateRequired", "%q in %q requires a new signing cert/key pair: %v", c.Name, c.Namespace, reason)
+		c.AdditionalAnnotations.RefreshPeriod = c.Refresh.String()
 		if err = setSigningCertKeyPairSecretAndTLSAnnotations(signingCertKeyPairSecret, c.Validity, c.AdditionalAnnotations); err != nil {
```

Comment:
let's pass `c.Refresh` to `setSigningCertKeyPairSecretAndTLSAnnotations` and then to `setTLSAnnotationsOnSigningCertKeyPairSecret` so that annotation assignments are in one place for the signer.

Reply:
actually, see https://github.com/openshift/library-go/pull/1971#discussion_r2228157172